src/devices/Gui/ScreenCapture.Windows.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Iot.Device.Graphics;
+
+namespace Iot.Device.Gui
+{
+    public partial class ScreenCapture
+    {
+        [SuppressMessage("Interoperability", "CA1416", Justification = "Only used on windows, see call site")]
+        private static unsafe BitmapImage GetScreenContentsWindows(Rectangle area)
+        {
+            try
+            {
+                using (Bitmap bitmap = new Bitmap(area.Width, area.Height, System.Drawing.Imaging.PixelFormat.Format32bppArgb))
+                {
+                    using (System.Drawing.Graphics g = System.Drawing.Graphics.FromImage(bitmap))
+                    {
+                        g.CopyFromScreen(new System.Drawing.Point(area.Left, area.Top), System.Drawing.Point.Empty, new System.Drawing.Size(area.Width, area.Height));
+                    }
+
+                    // For some reason, we need to swap R and B here. Strange...
```

Comment:
in most linux system, the colors are stored in BBGGRR and not RRGGBB like in Windows ;-)

Reply:
Ah, thanks for the explanation. Except that this is the Windows implementation that messes things up, it seems...

Reply:
Depending on the tools used, they may also store internally the "reverse" way.